studio/src/mas-side-nav.js

```diff
@@ -68,6 +69,13 @@ class MasSideNav extends LitElement {
         }
     `;
 
+    navigateToSplash() {
+        const url = new URL(window.location.href);
```

Comment:
with the state manager, the history/url management should be consolidated into a single location as a side-effect or reaction to state update.
I think it should be in `navigateToPage`.